kubernetes/kube.libsonnet

```diff
@@ -703,7 +702,8 @@
 
   LimitRange(name, namespace): $._Object('v1', 'LimitRange', name, namespace=namespace),
 
-  PodDisruptionBudget(name, namespace, app=name): $._Object('policy/v1beta1', 'PodDisruptionBudget', name, namespace=namespace) {
+  PodDisruptionBudget(name, namespace, app=name, version='policy/v1'): $._Object( version,
```

Comment:
Looks like this is the only change
```
an empty spec.selector ({}) written to a policy/v1 PodDisruptionBudget selects all pods in the namespace (in policy/v1beta1 an empty spec.selector selected no pods). An unset spec.selector selects no pods in either API version.
```
Probably just safe to update in place as we always specify a selector

Reply:
Agree, probably ok everywhere but possibly/likely breaks ops clusters